src/screens/SurveyDetail/question.js

```diff
@@ -31,7 +18,7 @@ const Questions = () => {
         <div className="questions__details">
           <div className="questions__number">1/5</div>
           <h1 className="questions__title"> { question }</h1>
-          <WheelSelect data={ data } multipleSelect={ true } />
+          <Rating initialRating={ 3 } ratingCount={ 5 } ratingEmoji="smiley" />
```

Comment:
How about `maxRating` instead of `ratingCount` 🤔  ?

Reply:
Fixed in [6feba1f](https://github.com/rafayet-monon/surveyor/pull/20/commits/6feba1f11e1136edeb3a544a514a0ee35dc1ebc7)